js/modules/tamagotchi.js

```diff
@@ -1,18 +1,36 @@
 export default class Tamagotchi {
   constructor() {
-    this.health = { value: 10, importance: 1 };
-    this.hunger = { value: 10, importance: 3 };
-    this.energy = { value: 10, importance: 2 };
-    this.fun = { value: 10, importance: 4 };
+    this.health = { value: 10, importance: 1, element: ".health" };
```

Comment:
**issue:** It should not be hardcoded. It won't work when you'd like to change your HTML structure. Check your `mount` method. You have these argument already - use them to set dynamically your parameters `element` value